### PR TITLE
Updates to IntroToKubernetes Ch03

### PR DESCRIPTION
**Pull request by larryclaman**
clean up structure.  Made references to using gui tools.

### CODE REVIEW

File: 001-IntroToKubernetes/Coach/03-k8sintro.md [hunk 1]
```diff
@@ -10,21 +10,23 @@
...
+  	- **NOTE:** Attaching an ACR requires the student to have Owner or Azure account administrator role on the Azure subscription. If this is not possible then someone who is an Owner can do the attach for the student after they create the cluster.
+		- See below for the `az aks update` command that is used to attach the ACR.
+	- Documentation on installing AKS can be found here:
 		- [Portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
 		- [CLI](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough)
 - It is usually a good idea to explain to the students what kind of options they have when creating a cluster. Doing a walkthrough demo of provisioning a cluster with the Portal is good showcasing tool, but end by telling them they need to figure out how to achieve the same thing with the CLI.
 - Have the teams show you the running cluster with:
 	- `kubectl get nodes`
-		- It should 3 nodes with each node in a different availability zone.  
+		- This show show three nodes, but it will not show the availability zone.
```
Comment (gfilicetti) [inline 001-IntroToKubernetes/Coach/03-k8sintro.md:23]: @larryclaman you have a slight typo in this sentence: "show show"

**Review by gfilicetti — CHANGES_REQUESTED** [review]
quick typo fix and we're done. FYI @jrzyshr .

File: 001-IntroToKubernetes/Coach/03-k8sintro.md [hunk 2]
```diff
@@ -10,21 +10,23 @@
...
+		- See below for the `az aks update` command that is used to attach the ACR.
+	- Documentation on installing AKS can be found here:
 		- [Portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
 		- [CLI](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough)
 - It is usually a good idea to explain to the students what kind of options they have when creating a cluster. Doing a walkthrough demo of provisioning a cluster with the Portal is good showcasing tool, but end by telling them they need to figure out how to achieve the same thing with the CLI.
 - Have the teams show you the running cluster with:
 	- `kubectl get nodes`
-		- It should 3 nodes with each node in a different availability zone.  
+		- This show show three nodes, but it will not show the availability zone.  
+    - To see the availability zone, run:  `kubectl get nodes -o custom-columns=NAME:'{.metadata.name}',REGION:'{.metadata.labels.topology\.kubernetes\.io/region}',ZONE:'{metadata.labels.topology\.kubernetes\.io/zone}'`
```
Comment (jrzyshr) [inline 001-IntroToKubernetes/Coach/03-k8sintro.md:24]: This is a spoonfeed IMO.  I think stick it in the coach's guide.

Personally, my go to here is to use 'kubectl describe node <nodename>' and let the students explore what data comes back.  They can google the fancy command that parses out just the zone #.  NO ONE will ever remember that as a takeaway from the hack! 

If you feel strongly about putting a pointer here, then go with 'describe node' and put a comment asking the student to review/explore the data that comes back.

**Review by jrzyshr — COMMENTED** [review]
Great stuff!  One minor request.

File: 001-IntroToKubernetes/Coach/03-k8sintro.md (hunk 2 above)
Comment (gfilicetti) [reply 001-IntroToKubernetes/Coach/03-k8sintro.md:24]: I agree with Peter... I don't see a reason to spoon feed them off the jump. Give it as a nugget to the coach to use if the students get stuck.

Comment (larryclaman) [reply 001-IntroToKubernetes/Coach/03-k8sintro.md:24]: Peter & Gino:  Maybe our changes are out of sync; The updates you're noting are only in the coach guide.

**Review by gfilicetti — APPROVED** [review]

**Comment by gfilicetti** [comment]
@larryclaman I'm good to go with your latest edits, and you're right the spoon feeding peter pointed out is already in the Coach's guide, sorry about that.

However you need to sync with upstream and resolve the conflicts that exist. After that i can push the button on this.

**Comment by larryclaman** [comment]
@gfilicetti conflicts resolved.  should be good to go.

**Pull request merged**